tests/combined/tst_combined_local.qml

```diff
@@ -81,4 +81,12 @@ TestCase {
     function test_signals() {
         Check.signals();
     }
+
+    property CombinedInterfaceUncreatable combiUncreatable
```

Comment:
We don't want a type called "XXXUncreatable"

Reply:
hmm, that's for the sake of testing. It makes it clear what is tested. What would you propose instead? I consider CombinedInterfaceUncreatable better than plain CombinedInterface here. The latter could be used, since the creatable type is called CombinedInterfaceAPI - which you dont like either, I know.